developer_manual/basics/controllers.rst

```diff
@@ -759,7 +764,7 @@ In order to ease migration from OCS API routes to the App Framework, an addition
 
 The format parameter works out of the box, no intervention is required.
```

Comment:
I know you didn't change this, but this sentence is a bit unclear. It talks about the `?format=json` parameter, but doesn't really explain it. IMO using the format parameter is not good, the `Accept: application/json` header should be used instead as it is standardized.

Reply:
I tried to address this in the latest commit.
![grafik](https://github.com/user-attachments/assets/ac4c6b2e-7f21-4ce4-a248-c6f124a96d23)